### ISSUE TITLE
IconFamily should default direction to LTR

### ISSUE THREAD

**Issue by joebochill** [pxblue/react-native-component-library#180]
#### Describe the bug
The default behavior when using the iconFamily format for specifying an icon (`{ name: '' }`) is auto, meaning that the icon will be flipped when the language is RTL.

#### What is the expected behavior?
Icons should not flip by default.

#### What are the steps to reproduce?
1. Open the showcase
2. Configure an icon using the icon family
3. Flip the direction to RTL
4. observe the icon is flipped
		
#### Anything else to add?
I'm logging this more as a question about what we want to be the default. If I'm not mistaken, I think the majority of icons _should not_ flip in RTL, meaning we should default to that behavior and have user identify just those icons that they _do_ want to flip to save them work.

**Comment by daileytj** [comment]
@huayunh What are your thoughts on the  "Anything else to add" section here?

**Comment by joebochill** [comment]
Technically, this would be a breaking change and would necessitate a new major version.

**Comment by huayunh** [comment]
Correct; most icons should not be flipped, with a few exceptions. For what to flip, see [here](https://brightlayer-ui.github.io/patterns/internationalization#bidirectionality--right-to-left-(rtl)-support)

**Comment by joebochill** [comment]
The prop for `direction` should default to 'ltr' instead of 'auto' so that most icons will not be flipped in RTL languages (only a handful of icons should be flipped).
